abel/tools/center.py

```diff
@@ -223,6 +250,41 @@ def find_center_by_center_of_mass(IM, verbose=False, round_output=False,
     return center
 
 
+def find_center_by_convolution(IM, **kwargs):
+    """ Center the image by convolution of two projections along each axis.
+
+        code from the ``linbasex`` juptyer notebook
+
+    Parameter
+    -------
+    IM: numpy 2D array
+        image data
+    Returns
+    -------
+    center: tuple
+        (row-center, col-center)
+
+    """
+    # projections along axis=0 of image (rows)
```

Comment:
Message may have been [here](https://github.com/PyAbel/PyAbel/pull/177#discussion_r63534948). You are correct, now fixed.